astropy/tests/runner.py

```diff
@@ -302,16 +302,23 @@ def package(self, package, kwargs):
             If nothing is specified all default Astropy tests are run.
         """
         if package is None:
-            self.package_path = self.base_path
+            self.package_path = [self.base_path]
         else:
-            self.package_path = os.path.join(self.base_path,
-                                        package.replace('.', os.path.sep))
-
-            if not os.path.isdir(self.package_path):
-                raise ValueError('Package not found: {0}'.format(package))
+            packages = package.split(',')
+
+            self.package_path = []
+            for package in packages:
+                package_path = os.path.join(self.base_path,
+                                            package.replace('.', os.path.sep))
+                if not os.path.isdir(package_path):
+                    warnings.warn('Package not found: {0}'.format(package))
```

Comment:
I'd just make this an error; the user surely does not want only a fraction of the tests done (and the warning will be gone before they know it).